README.md

```diff
@@ -58,7 +58,7 @@ make
 Run the following command
 
 ```
-python ./python/cuberite_process.py
+python3 ./python/cuberite_process.py
```

Comment:
we are using python3 throughout the project, and have mentioned it in the `Dependencies` section in the README.
As long as we upgrade to that this change shouldn't be necessary.